tests/testutils.py

```diff
@@ -42,22 +78,6 @@ def tearDown(self):
         from django_components.component import component_node_subclasses_by_name
         component_node_subclasses_by_name.clear()
 
-    # Mock the `generate` function used inside `gen_id` so it returns deterministic IDs
-    def _start_gen_id_patch(self):
```

Comment:
I moved this logic for the patching of the random ID generator out, so I could reuse it.